poofarmer/Scripts/Player.gd

```diff
@@ -1,16 +1,60 @@
-extends RigidBody2D
+extends Area2D
 
 
-# Declare member variables here. Examples:
-# var a = 2
-# var b = "text"
+signal playerFire(playerPosition, fireAngle)
 
+# Declare member variables here. Examples:
+export var speed = 300
+export var holdCapacity = 10
+var currentHoldAmount = 0
+var screen_size
```

Comment:
Since the player will be using the total poo for spending and ammo, I think it actually makes sense for the player script to manage the total.  It will look like the silo is holding the poo total but the reality is that it makes more sense on the player script since the player interacts with the total more often than the silo does.  This way the player script does not need to look up that poo total from the silo every time the player interacts with poo.